product-releases/2021-03-12-service-level-objectives.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Service Level Objectives
+tags: [service level objectives, standards]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<div class="avatar margin-vert--md minus-margin-top">
```

Comment:
this is not what we usually do for avatars, can you copy the ones from other blog posts (not the standards one)

Reply:
good catch, sure